api/ssi-bridge/src/tools/setup-credential/index.ts

```diff
@@ -0,0 +1,65 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
+import fs from 'fs';
+
+import { userService, verificationService } from '../../routers/services';
+import { ConfigurationService } from '../../services/configuration-service';
+import { Logger } from '../../utils/logger';
+import { MongoDbService } from '@iota/is-shared-modules/lib/services/mongodb-service';
+import { UserRoles, UserType } from '@iota/is-shared-modules/lib/models/types/user';
+import { CollectionNames } from '../../database/constants';
+import { CredentialTypes } from '@iota/is-shared-modules/lib/models/types/verification';
+
+/**
+ * Issues a credential from the root identity to a newly created identity and sets its role to admin
+ */
+const setupCredential = async () => {
+	try {
+		const configService = ConfigurationService.getInstance(Logger.getInstance());
+		const config = configService.config;
+
+		await MongoDbService.connect(config.databaseUrl, config.databaseName);
+
+		const rootIdentity = await configService.getRootIdentityId();
+		const adminIdentity = await userService.createIdentity({ username: `Admin-${Math.ceil(Math.random() * 100000)}` });
+
+		await createCredential(adminIdentity.doc.id, rootIdentity);
+		await setAdminRole(adminIdentity.doc.id);
+
+		fs.writeFileSync('./adminIdentity.json', JSON.stringify(adminIdentity, null, 4));
+		console.log('The following Identity can now issue credentials (it is also saved in the root directory as adminIdentity.json):');
+		console.log('Identity: ', { identityId: adminIdentity.doc.id, secretKey: adminIdentity.key.secret });
```

Comment:
Please do not log the secret key. Maybe this will be integrated into some build pipeline or so.